beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -4529,10 +4404,11 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
                 let block = self
                     .get_blinded_block(&parent_block_root)?
                     .ok_or(Error::MissingBeaconBlock(parent_block_root))?;
-                let state = self
-                    .get_state(&block.state_root(), Some(block.slot()))?
+                let (state_root, state) = self
```

Comment:
I think the log above needs to be updated to remove snapshot cache:

https://github.com/sigp/lighthouse/blob/970f3dfc6eca93bbdbf11b89e6c93157eb693bcd/beacon_node/beacon_chain/src/beacon_chain.rs#L4398-L4403